enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -1217,7 +1217,6 @@ class PerLearnerSpendCreditAccessPolicy(CreditPolicyMixin, SubsidyAccessPolicy,
 
     # Policies of this type *must not* define a per-learner enrollment limit or an assignment configuration
     FIELD_CONSTRAINTS = {
-        'assignment_configuration': (is_none, 'must not relate to an AssignmentConfiguration.'),
```

Comment:
@iloveagent57 Just an FYI that we are removing this constraint so a PerLearnerSpendCreditAccessPolicy can point to an assignment_configuration instance.